db/migrate/20170921110959_add_status_to_middleware_domain.rb

```diff
@@ -0,0 +1,8 @@
+class AddStatusToMiddlewareDomain < ActiveRecord::Migration[5.0]
+  class MiddlewareDomain < ActiveRecord::Base
+  end
```

Comment:
I think you want to disable STI here
`self.inheritance_column = :_type_disabled # disable STI`

Reply:
Could you please elaborate that a little bit?
I don't understand how the migration can change by disabling the STI

Reply:
@Fryguy can correct me if I'm wrong but I believe it is so we don't end up loading the models during the migration if the record is subclassed with STI

Reply:
`self.inheritance_column = :_type_disabled # disable STI`
here is when the table has column `type`. For more details see:https://github.com/ManageIQ/manageiq-schema/issues/4

Anyway, you don't need to define model here, as you are not using the model in migration either in spec.